frontend/src/metabase/actions/containers/ActionCreator/ActionContext/ActionContext.ts

```diff
@@ -15,9 +15,10 @@ export type ActionContextType = {
   isNew: boolean;
   isDirty: boolean;
   ui: ActionCreatorUIProps;
-  handleActionChange: (action: EditableActionParams) => void;
-  handleFormSettingsChange: (formSettings: ActionFormSettings) => void;
+  patchAction: (action: EditableActionParams) => void;
+  patchFormSettings: (formSettings: ActionFormSettings) => void;
   renderEditorBody: (props: EditorBodyProps) => React.ReactNode;
+  setAction: (action: WritebackAction) => void;
```

Comment:
Renamed `change` to `patch` to indicate a partial updates. Otherwise having both `change` and `set` would be confusing.